### ISSUE TITLE
Customize column width and behavior 

### ISSUE THREAD

**Issue by Eithcowich** [jbetancur/react-data-table-component#445]
I have tables with different type of data in each row, and I need to be able to customize each column's width. Something like the `css` way of doing things per column. Is it possible? 

I also have a column with text, and I'd like the column to stay at its width and wrap the text around. it's OK if the column becomes taller. 

Thanks for all your help!

**Comment by jbetancur** [comment]
Take a look at the column props: `width, minWidth, maxWidth` or for a more custom solution you can use `style`. You can also create a custom `cell`
https://github.com/jbetancur/react-data-table-component#161-columns

**Comment by Eithcowich** [comment]
Thanks @jbetancur, the params you point to are great and work for me. Is there also a way to get the content of the cell to wrap around, so that the cell expands vertically, and not horizontally?

**Comment by jbetancur** [comment]
I think this would just be css/style, so for example:
```
const ReallyLongEmail= row => (
  <a href="mailto://" style={{ wordBreak: "break-all" }}>
    {row.email}
  </a>
);

....

  {
    name: "EmailWrapped",
    selector: "email",
    sortable: true,
    cell: row => <ReallyLongEmail{...row} />,
    maxWidth: "150px"
  },

```

**Comment by jbetancur** [comment]
You cal also try the column.wrap property if you do not need a custom `cell` and combine that with maxWidth.

This is an older code sandbox with some wrapping examples then I have not gotten around to publishing but may be helpful

https://codesandbox.io/s/react-data-table-sandbox-custom-growwtap-9upub

**Comment by jbetancur** [comment]
Even better you can just do this with `style` and `wrap`

```
  {
    name: "My Field",
    selector: "field",
    sortable: true,
...
    wrap: true,
    style: {
      wordBreak: 'break-all'
    },
  },
```

**Comment by jbetancur** [comment]
Wow, this is actually a bug that wrap requires the extra style! wordBreak should default to break-word and not require the style.  I will get this fixed. In the meantime, you can use the workaround above

**Comment by jbetancur** [comment]
This was a quick fix - update your RDT to version 6.2.3 and column[].wrap should work as intended without any additional style or custom cell

**Comment by Eithcowich** [comment]
Great, thanks @jbetancur!
